src/services/Program.cpp

```diff
@@ -83,13 +83,16 @@ void Program::loadGraphs(int *percentage) {
 
     *percentage=0;
     this->graphs[0] = Graph();
-    Scrapper().scrape(this->graphs[0], "../files/real_graphs/graph1/nodes.csv", "../files/real_graphs/graph1/edges.csv",0);
+    //Scrapper().scrape(this->graphs[0], "../files/real_graphs/graph1/nodes.csv", "../files/Extra_Fully_Connected_Graphs/edges_25.csv",1);
     //this->graphs[0].completeRealEdges();
+    this->graphs[0] = Graph();
```

Comment:
Why do you have this line here at line 88 if you are already defining it in line 85?